recipes/newrelic/infrastructure/amazonlinux.yml

```diff
@@ -142,6 +142,5 @@ postInstall:
       ⚙️  The Infrastructure Agent configuration file can be found in /etc/newrelic-infra.yml
       Edit this file to make changes or configure advanced features for the agent. See the docs for options:
       https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings
-      
-      Note: Process monitoring has been enabled by default - all other config options are left to the user.
 
+      Note: Process monitoring has been enabled by default - all other config options are left to the user.
```

Comment:
I'd be careful about removing the indentation on this empty line. Might be invalid yaml without it.

Reply:
My auto formatter did this

Reply:
And yep, I think it's causing an issue. Thanks